Paintroid/src/main/java/org/catrobat/paintroid/iotasks/OpenRasterFileFormatConversion.java

```diff
@@ -94,21 +98,24 @@ public static void setContext(MainActivity toSet) {
 		mainActivity = toSet;
 	}
 
-	public static Uri exportToOraFile(List<Bitmap> bitmapList, String fileName, Bitmap bitmapAllLayers, ContentResolver resolver) throws IOException {
+	public static Uri exportToOraFile(List<LayerContracts.Layer> layers, String fileName, Bitmap bitmapAllLayers, ContentResolver resolver) throws IOException {
```

Comment:
Did we make sure that this is backwards compatible for old ora files?

Reply:
Tested and works.